src/command_ui.cc

```diff
@@ -584,4 +584,6 @@ initialize_command_ui() {
 
   CMD2_ANY_LIST ("elapsed.less",         std::bind(&apply_elapsed_less, std::placeholders::_2));
   CMD2_ANY_LIST ("elapsed.greater",      std::bind(&apply_elapsed_greater, std::placeholders::_2));
+
+  CMD2_VAR_STRING("ui.status.throttle_up_name", "");
```

Comment:
This should be 'ui.status.throttle.up.name', and should also include the option for the down throttle.